modules/packages/LAPACK.chpl

```diff
@@ -19094,47 +19094,47 @@ Wrapped procedure of LAPACKE_sbdsdc for the type real(32).
  */
```

Comment:
Note: This file is generated by a script, so this will be a deviation from the script-generated code. I think we have already deviated from the script-generated code though, so that probably doesn't matter.

Reply:
Yeah, I always wince when I make changes like this, then come to the same conclusion you did, and do it anyway.  I figure on the day someone needs to run the script again, their first step will be to update the script to generate the current file (or something close to it).